diff --git a/php/src/Payment/Common/Client.php b/php/src/Payment/Common/Client.php
index 82afe06..3d01b02 100644
--- a/php/src/Payment/Common/Client.php
+++ b/php/src/Payment/Common/Client.php
@@ -218,7 +218,7 @@ class Client {
      * @throws Exception
      * @throws TeaUnableRetryError
      */
-    public function refund($outTradeNo, $refundAmount){
+    public function refund($outTradeNo, $refundAmount, $outRequestNo = null){
         $_runtime = [
             "ignoreSSL" => $this->_kernel->getConfig("ignoreSSL"),
             "httpProxy" => $this->_kernel->getConfig("httpProxy"),
@@ -258,6 +258,9 @@ class Client {
                     "out_trade_no" => $outTradeNo,
                     "refund_amount" => $refundAmount
                 ];
+                if ($outRequestNo !== null) {
+                    $bizParams["out_request_no"] = $outRequestNo;
+                }
                 $textParams = [];
                 $_request->protocol = $this->_kernel->getConfig("protocol");
                 $_request->method = "POST";
